nntrainer/tensor/tensor_v2.h

```diff
@@ -245,6 +245,35 @@ class TensorV2 {
    */
   bool operator!=(const TensorV2 &rhs) const { return !(*this == rhs); }
 
+  /**
+   * @brief Construct a new Tensor object from a buffer
+   * This will not copy buffer to a new tensor but directly uses it
+   *
+   * @param[in] buf buffer
+   * @param[in] bytes buffer size in bytes
+   * @param[in] d tensor dim
+   * @param[in] offset offset to be used from current
+   * @return    Tensor object
+   * @throws    std::invalid_argument if buf is null
```

Comment:
Adding a note that the buffer is not owned by the mapped tensor would be better.
the user may be confused.

Reply:
applied!